doc/user_doc/vic_installation/security.md

```diff
@@ -0,0 +1,72 @@
+# Securing Virtual Container Host Connections #
+
+vSphere Integrated Containers Engine provides different possibilities for securing the connections between virtual container hosts and different elements in your environment.
+
+- [Connections to Docker clients](#docker)
+- [Connections to Registry Servers for Container Images](#registries)
+- [Connections to and Downloads from the VCH Admin Portal](#vicadmin)
+- [Connections to vSphere](#vsphere)
+
+<a name="docker"></a>
+## Connections to Docker Clients ##
+
+Docker uses TLS authentication to allow for different levels of security when connecting Docker clients to container hosts. For information about Docker certificates and security, see [the Docker security documentation](https://docs.docker.com/engine/security/https/).
+
+vSphere Integrated Containers Engine allows you to deploy virtual container hosts that implement the following standard Docker authentication options:
+
+- Two-Way TLS authentication with trusted auto-generated or custom certificates
```

Comment:
I'd simplify this set:
* mutual authentication with client and server certificates
* server authentication, no client authentication
* no TLS authentication